cmd/migrate.go

```diff
@@ -32,10 +32,17 @@ func init() {
 		Long:  `Upgrade your database to be compatible with this service broker.`,
 		Run: func(cmd *cobra.Command, args []string) {
 			logger := lager.NewLogger("migrations-cmd")
+			logger.RegisterSink(lager.NewWriterSink(os.Stderr, lager.DEBUG))
+
+			logger.Debug("Setting up the database")
 			db := db_service.SetupDb(logger)
+
+			logger.Debug("Starting the migration")
 			if err := db_service.RunMigrations(db); err != nil {
-				log.Fatalf("Error running migrations: %v\n", err)
+				logger.Fatal("Error running migrations", err)
```

Comment:
I wonder if it might be appropriate to return from this block. Otherwise the subsequent "Finished migration" debug log will appear which might be confusing.

Reply:
Based on what I saw, this should do an exit like log.Fatal but it's not very clear. I'm also not 100% convinced we should be using this custom logger rather than go's builtin. It's being used inconsistently and it's yet another dependency.

Reply:
+1 to questioning the need for `lager`. I'll add an issue to address that separately.